catalog/camel-catalog/src/generated/resources/org/apache/camel/catalog/docs/kafka-component.adoc

```diff
@@ -580,3 +580,23 @@ from("kafka:my_topic?headerFilterStrategy=#myStrategy")
 `myStrategy` object should be subclass of `HeaderFilterStrategy` and must be placed in the Camel registry, either manually or by registration as a bean in Spring/Blueprint, as it is `CamelContext` aware.
 
 include::camel-spring-boot::page$kafka-starter.adoc[]
+
+== Consuming Avro messages from Kafka using the schema-registry
+*Since Camel 3.0.0*
+
+This route applies the schema and converts the Avro messages to POJOs for further processing:
+
+[source,java]
+----
+from(kafka("avro-events")
+    .brokers("localhost:9092")
+    .schemaRegistryURL("http://localhost:8081")
+    .specificAvroReader(true)
+    .valueDeserializer("io.confluent.kafka.serializers.KafkaAvroDeserializer")
```

Comment:
Is it only confluent that has avra deseriazlier and this implies you are using confluent schema registry. Can we make this example also work with other schema registries? And add some more notes on this? And btw the example is using endpoint-dsl which we should mention.

Reply:
I am all for improving it, but please note that I am totally lacking on the required expertise. All I did in this PR is just summing up the info gathered by @tstuber in https://github.com/apache/camel-quarkus/issues/1960 .

Reply:
Actually, the adjusted code misses now a closing bracket at the end: `.valueDeserializer("io.confluent.kafka.serializers.KafkaAvroDeserializer"))`